rs/tests/driver/Cargo.toml

```diff
@@ -27,6 +27,7 @@ ed25519-dalek = { workspace = true }
 flate2 = { workspace = true }
 futures = { workspace = true }
 humantime = "2.0"
+humantime-serde = "1.1.1"
```

Comment:
Could you add this to the workspace? We recently added a [Rust Dependency Policy](https://github.com/dfinity/ic/blob/master/README.adoc#rust-dependency-policy) that requires this.